scripts/ci/pylint_todo.txt

```diff
@@ -198,34 +196,6 @@
 ./airflow/ti_deps/deps/task_concurrency_dep.py
 ./airflow/ti_deps/deps/trigger_rule_dep.py
 ./airflow/ti_deps/deps/valid_state_dep.py
-./airflow/utils/asciiart.py
-./airflow/utils/cli_action_loggers.py
-./airflow/utils/compression.py
-./airflow/utils/configuration.py
-./airflow/utils/dates.py
-./airflow/utils/db.py
-./airflow/utils/decorators.py
-./airflow/utils/email.py
-./airflow/utils/file.py
-./airflow/utils/helpers.py
-./airflow/utils/json.py
-./airflow/utils/log/es_task_handler.py
-./airflow/utils/log/file_processor_handler.py
-./airflow/utils/log/gcs_task_handler.py
-./airflow/utils/log/logging_mixin.py
-./airflow/utils/log/s3_task_handler.py
-./airflow/utils/log/wasb_task_handler.py
-./airflow/utils/module_loading.py
-./airflow/utils/net.py
-./airflow/utils/operator_helpers.py
-./airflow/utils/operator_resources.py
-./airflow/utils/sqlalchemy.py
-./airflow/utils/state.py
-./airflow/utils/tests.py
-./airflow/utils/timeout.py
-./airflow/utils/timezone.py
-./airflow/utils/trigger_rule.py
-./airflow/utils/weight_rule.py
```

Comment:
These changes shouldn't be included in your PR.

Reply:
Fixed. rebased master